could not determine kind of name for C.free

On windows when I'm installing via the following command as specified in the readme (after downloading the libvips and setting it all up)

```
$ go get -u -v github.com/davidbyttow/govips/v2/vips
```

```
go: found github.com/davidbyttow/govips/v2/vips in github.com/davidbyttow/govips/v2 v2.0.1
go: golang.org/x/text upgrade => v0.3.4
go: golang.org/x/net upgrade => v0.0.0-20201027133719-8eef5233e2a1
github.com/davidbyttow/govips/v2/vips
# github.com/davidbyttow/govips/v2/vips
C:\Users\Rithvij\go\pkg\mod\github.com\davidbyttow\govips\v2@v2.0.1\vips\lang.go:12:2: could not determine kind of name for C.free
```

I was able to fix it by adding `// #include <stdlib.h>` in line 3 of lang.go and building again `go build -v github.com/davidbyttow/govips/v2/vips`

Can it be fixed? Seems like a simple fix.

--- a/vips/lang.go
+++ b/vips/lang.go
@@ -2,6 +2,7 @@ package vips
 
 // #cgo pkg-config: vips
 // #include <vips/vips.h>
+// #include <stdlib.h>
 import "C"
 
 import (

